src/intra_edge.rs

```diff
@@ -324,17 +323,21 @@ impl<const SB128: bool, const N_BRANCH: usize, const N_TIP: usize>
 
         let sb128 = SB128 as u8;
 
-        let mut bl = BL_128X128;
-        while bl <= BL_32X32 {
-            indices.branch[bl as usize].index = level_index(bl + sb128);
+        let mut bl = BlockLevel::Bl128x128 as u8;
+        while bl <= BlockLevel::Bl32x32 as u8 {
+            indices.branch[bl as usize].index = level_index(bl as u8 + sb128);
             bl += 1;
         }
 
-        let bl = if SB128 { BL_128X128 } else { BL_64X64 };
+        let bl = if SB128 {
+            BlockLevel::Bl128x128
+        } else {
+            BlockLevel::Bl64x64
+        };
         (self, indices) = self.init_mode_node(EdgeIndex::root(), bl, indices, true, false);
 
-        let mut bl = BL_128X128;
-        while bl <= BL_32X32 {
+        let mut bl = BlockLevel::Bl128x128 as u8;
+        while bl <= BlockLevel::Bl32x32 as u8 {
```

Comment:
so sad this can't be a `for bl in [BlockLevel::Bl128x128, BlockLevel::Bl64x64, BlockLevel::Bl32x32]`. one day...